Homework02/main.py

```diff
@@ -0,0 +1,38 @@
+"""
+First modul for homework 02
+
+Created: 01.11.21, 19:17
+
+Author: LDankert
+"""
+
```

Comment:
Sorry, for only commeting now. I looked through i earlier, but I have a cold so my concentration level is not the best today to give a good feedback